client/containers/Pub/PubSyncManager.js

```diff
@@ -212,6 +215,7 @@ class PubSyncManager extends React.Component {
 				if (this.state.firebaseRootRef && hasUpdates) {
 					this.state.firebaseRootRef.child('metadata').update(firebaseSyncData);
 				}
+				document.title = getPubPageTitle(this.state.pubData, this.props.communityData);
```

Comment:
What is the need for having the `document.title = get...` line both on line 178 and line 218? Is there a case where those two are setting different values?

Reply:
Ah, this was a mistake. My bad!